### PR TITLE
Go modules support

### PR DESCRIPTION
**Pull request by asahasrabuddhe**
In line with Go's release policy, dropping support for Go 1.10 and adding support for Go 1.13

### CODE REVIEW

**Comment by asahasrabuddhe** [comment]
At a very interesting point here:

This PR would never have all the test cases pass. There are two things going on here:

1. The build fails at the gfmrun stage where the gfmrun is not able to successfully run two of the README.md examples reference ShortOptionHandling (in master) which is not available in the latest release 1.21.0 of the cli.

2. Support I were to remove these references from the code and push, the gfmrun stage would pass but the build would again fail at the git diff stage as the README.md pushed with the PR would again not match the README.md at the master.

This brings me to the issue: #884

**Comment by lynncyrin** [comment]
> two of the README.md examples reference ShortOptionHandling (in master) which is not available in the latest release 1.21.0 of the cli

I have a PR for releasing `1.22.0` with `ShortOptionHandling ` here => https://github.com/urfave/cli/pull/882

File: .travis.yml [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
   - 1.11.x
   - 1.12.x
+  - 1.13.x
 
 os:
   - linux
   - osx
 
+env:
+  GO111MODULE=on
```
Comment (saschagrunert) [inline .travis.yml]: do we want to set `GOPROXY=https://proxy.golang.org` to speed things up?

Comment (asahasrabuddhe) [reply .travis.yml]: Good Idea! I must have missed it. I have added it now.

**Comment by asahasrabuddhe** [comment]
This PR is based of the original flags PR.

**Comment by asahasrabuddhe** [comment]
I can remove that and separate it if you like

**Comment by AudriusButkevicius** [comment]
Right, either lets make the changes completely independent, or not file them at the same time if they are, and wait for required change to land.

**Comment by asahasrabuddhe** [comment]
Fair enough. I will clean this up

**Comment by asahasrabuddhe** [comment]
I have rebased this and cleaned up.

**Comment by asahasrabuddhe** [comment]
This build will fail until such time that 1.22.0 is merged and UseShortHandOptions is available for the tests to succeed.

**Comment by asahasrabuddhe** [comment]
All green here too :)

**Pull request merged**